decorator.rb

```diff
@@ -0,0 +1,12 @@
+require './nameable'
+
+class Decorator < Nameable
```

Comment:
- Hey, since you used `@decoration`, please, kindly add it to the attr_accessor `attr_accessor :decoration`, or better still, you can use `nameable` for readability purposes. Add it before the constructor.